README.md

```diff
@@ -8,7 +8,10 @@ Unvanquished support files for map editors.
 Using the support files
 -----------------------
 
-Pick the directory corresponding to your editor and merge the file hierarchy inside the `gamepack` subdirectory into its installation. Further install instructions are given in a seperate per-editor readme file.
+Pick the directory corresponding to your editor and merge the file hierarchy inside the `gamepack` subdirectory into its installation. Further installation instructions are given in a seperate per-editor readme file.
+
+* [GtkRadiant readme](build/gtkradiant/README.md)
```

Comment:
Could also keep the readme files in `src` or in a `doc` folder and copy them into `build` for consistency.